site/content/en/v1.4/tasks/operations/airgap-deployment.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "Deploy Envoy Gateway in Air-Gapped Environments"
+---
+
+Deploying the Envoy Gateway in an air-gapped environment using a Helm chart requires careful configuration of the `values.yaml` file as well as adjustments when deploying a Gateway resource.
+
+You will need to specify custom image repositories for the following components in the Helm chart:
+
+- Gateway
+- Ratelimit
+
+## Gateway – `values.yaml` Configuration
+
+```yaml
+deployment:
+  envoyGateway:
+    image:
+      repository: custom-cr.internal.io/envoyproxy/gateway
+      tag: v1.4.1
+```
+
+## Ratelimit - `values.yaml` Configuration
+
+```yaml
+global:
+  images:
+    ratelimit:
+      image: custom-cr.internal.io/envoyproxy/ratelimit:master
+```
```

Comment:
Can we also mention the imagePullSecrets setting? ImagePullSecrets would be needed if the registry is private.